addons/l10n_bg/data/tax_report.xml

```diff
@@ -320,9 +320,94 @@
+                        </field>
+                    </record>
+                </field>
+            </record>
+
+            <record id="l10n_bg_tax_report_d" model="account.report.line">
+                <field name="name">Section D. VAT for deposition</field>
+                <field name="children_ids">
+                    <record id="l10n_bg_tax_report_70" model="account.report.line">
+                        <field name="name">[70] Import tax from Article 50, deducted in accordance with Art. 92, para. 1 VAT</field>
+                        <field name="code">BG_TR_70</field>
+                        <field name="expression_ids">
+                            <record id="l10n_bg_tax_report_refund" model="account.report.expression">
+                                <field name="label">refund</field>
+                                <field name="date_scope">from_beginning</field>
+                                <field name="engine">account_codes</field>
+                                <field name="formula">4531 + 4532 + 4538 + 4539</field>
+                            </record>
+
+                            <record id="l10n_bg_tax_report_refund_remaining" model="account.report.expression">
+                                <field name="label">refund_remaining</field>
+                                <field name="engine">aggregation</field>
+                                <field name="formula">BG_TR_70.refund</field>
                                 <field name="subformula">if_below(BGN(0))</field>
                             </record>
+
+                            <record id="l10n_bg_tax_report_70_tag" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">aggregation</field>
+                                <field name="formula">BG_TR_70.refund_remaining + BG_TR_50.balance</field>
+                                <field name="subformula">if_above(BGN(0))</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="l10n_bg_tax_report_71" model="account.report.line">
+                        <field name="name">[71] Input tax from cl. 50, effectively input</field>
+                        <field name="code">BG_TR_71</field>
+                        <field name="expression_ids">
+                            <record id="l10n_bg_tax_report_71_tag" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">aggregation</field>
+                                <field name="formula">BG_TR_50.balance - BG_TR_70.balance</field>
+                            </record>
+                        </field>
+                    </record>
+                </field>
+            </record>
+
+            <record id="l10n_bg_tax_report_e" model="account.report.line">
```

Comment:
What's the use of this section? 
The user can input any value he wants and isn't ever used?
Isn't there a system of postponing VAT refund/credit from a period to another to avoid making payments each period and so a use for it?? We have that in quite a few localizations (if not most of them)

Reply:
The second part of my question was answered with my tests of enterprise, but not about the use of the section :smiley: